### DISCUSSION TITLE
Worried about no new release version

### DISCUSSION THREAD

**Discussion by thingsiplay** [qtile/qtile#4376] [General]
It's been almost a year since last official release version. Why is there no new version? Mind you, this not pressuring anyone, I'm just genuinely asking. Many people on, me included nowadays, do not use the Git version but the traditional package from their distribution. And some are worried and ask if it should be forked or switched to another window manager. I don't see any apparent reason and the development itself is going good, as always. Is there something that blocks it?

**Comment by elParaguayo** [comment]
Rest assured there is a lot of development going on!

There are numerous reasons why there hasn't been a new release for a while - the main one is that our lives got in the way and we needed to focus on some other things.

That being said, I think we are *very* close to the next release.

**Comment by thingsiplay** [comment]
I know about active development, as I look into the issue tracker or commits sometimes to look what is happening currently. :-) That's why I was wondering in the first place. Apologies if my question comes of as offensive or pressuring in any way. Glad everything is okay, including the docs working again! Thank you for the quick response.

**Reply by elParaguayo** [reply]
No pressure at all. This release has taken much longer than we would have liked and it's totally reasonable to ask why.

**Comment by ExpandingMan** [comment]
Any news about this?  For most things I'd be more happy to just try out master, but for a major component like a window manager I much prefer to use the main manjaro package, which means release version.  I doubt I'm alone in feeling this way about it.

**Reply by elParaguayo** [reply]
It's a fair question. To be honest, I think all the devs are a bit frustrated with this at the moment! We're trying to sort out the packaging and it's proving a little bit challenging with the changes to how pip builds things in isolated environments etc etc. 

We're also seeing test failures happen from changes to different packages so we also need to look into these.

We're not looking to add any more features for the next release but we're just not ready to release yet.

**Comment by m-col** [comment]
New version is released :)

**Comment by thingsiplay** [comment]
Thank you. This is great news! (should this discussion topic be closed?)

Qtile 0.23.0, released 2023-09-24:
https://github.com/qtile/qtile/blob/master/CHANGELOG
